### PR TITLE
Upgrade RN for Kommander 2.2 including scope for k8s 1.22

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-84943

## Description of changes being made
Copied RN of 2.1 to 2.2 and added info on support for k8s 1.22
Replaced Konvoy/Kommander mentions with DKP, when possible

### Preview
You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4130.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 1]
```diff
@@ -2,10 +2,185 @@
...
+
+#### DKP major version upgrade
+
+Version 2.2 represents a major upgrade that moves forward D2IQ's architecture to give you access to the next generation of centralized Kubernetes and smart cloud native applications. It incorporates ClusterAPI as a major re-architecture in its management of production Kubernetes clusters. [ClusterAPI](https://cluster-api.sigs.k8s.io/introduction.html), or CAPI, enables declarative creation, configuration, and management of clusters. Declarative mode is a Kubernetes best practice that simplifies lifecycle tasks, making them more consistent and more repeatable. 2.2 enhances your existing clusters to use a new architecture.
+
+For more information on planning your upgrade, see the [DKP Major Version upgrade](/dkp/konvoy/2.2/major-version-upgrade/) guide.
+
+#### Provision Kubernetes Infrastructure from DKP
+
+Version 2.2 allows for provisioning and managing of Kubernetes clusters, making it easier than ever to get your infrastructure up and running quickly.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/release-notes/index.md:35]:
```suggestion
This version allows for provisioning and managing of Kubernetes clusters, making it easier than ever to get your infrastructure up and running quickly.
```

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 2]
```diff
@@ -2,10 +2,185 @@
...
+
+[Download](../../download/) and [install](../../install/) the latest version to get started.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download Kommander. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install this product.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in DKP.
+
+Kommander 2.2 supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster you want to attach using Kommander 2.1 must be running a Kubernetes version in that range.
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/release-notes/index.md:23]: Just for later, we could link this to the upgrade Kubernetes documentation we will create.

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 3]
```diff
@@ -2,10 +2,185 @@
...
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in DKP.
+
+Kommander 2.2 supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster you want to attach using Kommander 2.1 must be running a Kubernetes version in that range. 
+
+### New features and capabilities
+
+#### DKP major version upgrade
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/release-notes/index.md:27]: Take this whole section out. Major version upgrade was ONLY for 2.1. We won't support this anymore in 2.2 and have already removed the topics that tell people how to do this.

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 4]
```diff
@@ -2,10 +2,185 @@
...
+
+For more information on planning your upgrade, see the [DKP Major Version upgrade](/dkp/konvoy/2.2/major-version-upgrade/) guide.
+
+#### Provision Kubernetes Infrastructure from DKP
+
+This version allows for provisioning and managing of Kubernetes clusters, making it easier than ever to get your infrastructure up and running quickly.
+
+For more information on provisioning, see [Managing Clusters](../../clusters/).
+
+#### Continuous deployment
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/release-notes/index.md:39]: Let's remove every section in this heading. All of these features were for 2.1. So what we need is just one prompt that says #### Feature 1 to remind people where to put info later.

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 5]
```diff
@@ -2,10 +2,185 @@
...
+- project-grafana-loki: 0.33.1
+- project-logging: 1.0.0
+- prometheus-adapter: 2.11.1
+- reloader: 0.0.99
+- thanos: 0.4.5
+- traefik: 10.3.0
+- traefik-forward-auth: 0.3.2
+- velero: 3.1.3
+
+## Known issues
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/release-notes/index.md:86]: leave known issues as a heading, but remove all the things below. let's hope we don't have any known issues in 2.2.

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 6]
```diff
@@ -2,10 +2,185 @@
...
+
+Version 2.2 supports continuous delivery/deployment using Flux, which is designed for Kubernetes and supports multi-cluster and multi-tenant use cases. Configure Projects with GitOps-based Continuous Deployments using FluxCD, which enables canary and A/B deployments, as well as roll-back. DKP now uses a completely declarative approach, where what you define for production is what you get, without the need to monitor and manually intervene when something goes wrong.
+
+For more information on setting up continuous deployment using Flux, see [Continuous deployment](../../projects/project-deployments/continuous-delivery).
+
+#### Licensing through Amazon Marketplace
+
+You can purchase a license for DKP through the AWS Marketplace. In the UI, you can see information such as the license status (valid or expired), the license capacity (number of cores or clusters), and expiration date.
+
+### Component updates
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/release-notes/index.md:49]: We can clear out all the components in this section for now.

File: pages/dkp/kommander/2.2/release-notes/index.md (hunk 3 above)
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/release-notes/index.md:27]:
```suggestion
```

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 7]
```diff
@@ -2,10 +2,185 @@
...
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in DKP.
+
+Kommander 2.2 supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster you want to attach using Kommander 2.1 must be running a Kubernetes version in that range. 
+
+### New features and capabilities
+
+#### DKP major version upgrade
+
+Version 2.2 represents a major upgrade that moves forward D2IQ's architecture to give you access to the next generation of centralized Kubernetes and smart cloud native applications. It incorporates ClusterAPI as a major re-architecture in its management of production Kubernetes clusters. [ClusterAPI](https://cluster-api.sigs.k8s.io/introduction.html), or CAPI, enables declarative creation, configuration, and management of clusters. Declarative mode is a Kubernetes best practice that simplifies lifecycle tasks, making them more consistent and more repeatable. 2.2 enhances your existing clusters to use a new architecture.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/release-notes/index.md:29]:
```suggestion

```

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 8]
```diff
@@ -2,10 +2,185 @@
...
+
+For more information on planning your upgrade, see the [DKP Major Version upgrade](/dkp/konvoy/2.2/major-version-upgrade/) guide.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/release-notes/index.md:31]:
```suggestion

```

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 9]
```diff
@@ -2,10 +2,185 @@
...
+
+#### Provision Kubernetes Infrastructure from DKP
+
+This version allows for provisioning and managing of Kubernetes clusters, making it easier than ever to get your infrastructure up and running quickly.
+
+For more information on provisioning, see [Managing Clusters](../../clusters/).
+
+#### Continuous deployment
+
+Version 2.2 supports continuous delivery/deployment using Flux, which is designed for Kubernetes and supports multi-cluster and multi-tenant use cases. Configure Projects with GitOps-based Continuous Deployments using FluxCD, which enables canary and A/B deployments, as well as roll-back. DKP now uses a completely declarative approach, where what you define for production is what you get, without the need to monitor and manually intervene when something goes wrong.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/release-notes/index.md:41]:
```suggestion

```

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 10]
```diff
@@ -2,10 +2,185 @@
...
+
+For more information on setting up continuous deployment using Flux, see [Continuous deployment](../../projects/project-deployments/continuous-delivery).
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/release-notes/index.md:43]:
```suggestion
```

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 11]
```diff
@@ -2,10 +2,57 @@
...
+
+[Download](../../download/) and [install](../../install/) the latest version to get started.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download Kommander. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install this product.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in DKP.
+
+This version supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster you want to attach using Kommander 2.1 must be running a Kubernetes version in that range.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/release-notes/index.md:23]: Casie's comment got deleted: "Just for later, we could link this to the upgrade Kubernetes documentation we will create."

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 12]
```diff
@@ -2,10 +2,57 @@
...
+
+For more information on provisioning, see [Managing Clusters](../../clusters/).
+
+#### Feature 1, 2, etc.
+
+
+
+#### Licensing through Amazon Marketplace
+
+You can purchase a license for DKP through the AWS Marketplace. In the UI, you can see information such as the license status (valid or expired), the license capacity (number of cores or clusters), and expiration date.
```
Comment (cneth) [inline pages/dkp/kommander/2.2/release-notes/index.md:39]: I am not sure the second sentence adds a lot here.  Also, while technically accurate, customers cannot actually purchase 'cluster' licenses from AWS as we stopped offering those licenses just as this feature was developed.   In other words, it's confusing to mention 'number of ... clusters' here as no customer has this type of license from AWS, nor can they purchase this type of license.

```suggestion
You can purchase a license for DKP through the AWS Marketplace. After adding the license in the UI, you can see information such as the license status, the license capacity, and expiration date.
```

**Review by cneth — APPROVED** [review]

File: pages/dkp/kommander/2.2/release-notes/index.md [hunk 13]
```diff
@@ -2,10 +2,57 @@
...
+
+For more information on provisioning, see [Managing Clusters](../../clusters/).
+
+#### Feature 1, 2, etc.
+
+
+
+#### Licensing through Amazon Marketplace
+
+You can purchase a license for DKP through the AWS Marketplace. After adding the license in the UI, you can see information such as the license status, the license capacity, and expiration date.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/release-notes/index.md:39]:
```suggestion
You can purchase a license for DKP through the AWS Marketplace. Add the license in the UI, and you will see information such as the license status, the license capacity, and expiration date.
```

**Comment by msdolbey** [comment]
@gvasquezvargas - I fixed some links in here, so you'll want to run
`git fetch`
`git pull`
before you make any new changes

**Review by mwrightd2 — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

**Pull request merged**